README.md

```diff
@@ -201,6 +201,20 @@ name = "Tom"
 name = "Pradyun"
 ```
 
+Since bare keys are allowed to compose of only ASCII integers, it is possible
+to write dotted keys that look like floats but are 2-part dotted keys. Don't do
+this unless you have a good reason to (you probably don't).
+
+```toml
+3.14159 = "pi"
```

Comment:
How about add `"3.141592" = "pi"`?  By contrast, we can make it clearer.

Reply:
I wanted to make it clear what value came from what key. :)

Reply:
👌